lldb/test/API/CMakeLists.txt

```
@@ -58,8 +58,13 @@ set(LLDB_TEST_COMMON_ARGS
   -S nm
   -u CXXFLAGS
   -u CFLAGS
+  )
+if (LLDB_ENABLE_SWIFT_SUPPORT)
+  set(LLDB_TEST_COMMON_ARGS
+  ${LLDB_TEST_COMMON_ARGS}
   ${SWIFT_TEST_ARGS}
```

use `list(APPEND VAR ...)`?

```suggestion
  list(APPEND LLDB_TEST_COMMON_ARGS ${SWIFT_TEST_ARGS})
```

---

That is nicer but I completely removed my change (see Jonas comment).